services/web-server/src/graphql/Tasks.graphql

```diff
@@ -179,102 +179,6 @@ type Task {
-  # their resolution is.
-  requires: TaskRequire
-
-  # List of task-specific routes. AMQP messages will be CC'd to these routes.
-  routes: [String]
-
-  # Priority of task. this defaults to `LOWEST` and the scope
-  # `queue:create-task:<priority>/<provisionerId>/<workerType>` is required
-  # to define a task with `<priority>`.
-  priority: TaskPriority
-
-  # Number of times to retry the task in case of infrastructure issues.
-  # An _infrastructure issue_ is a worker node that crashes or is shutdown.
-  retries: Int
-
-  # Creation time of task.
-  created: DateTime!
-
-  # Deadline of the task. `pending` and `running` runs are resolved as **exception**
-  # if not resolved by other means before the deadline. _Note: deadline cannot be
-  # more than 5 days into the future._
-  deadline: DateTime!
-
-  # Time at which task definition and status are expired and deleted. Notice that
-  # all artifacts for the task must have an expiration that is no later than this.
-  # If this property isn't set, it will be set to `deadline` plus one year.
-  # This default may subject to change.
-  expires: DateTime
-
-  # List of scopes or scope patterns that the task is authorized to use.
-  scopes: [String]
-
-  # Task-specific payload following a worker-specific format. For example,
-  # `docker-worker` requires keys like: `image`, `commands`, and `features`.
-  # Refer to the documentation of `docker-worker` for details.
-  payload: JSON!
-
-  # Required task metadata.
-  metadata: TaskMetadataInput!
-
-  # Arbitrary key-value tags. Only accepts strings limited to 4KB.
-  # These can be used to attach arbitrary metadata to a task.
-  # Use this for informal tags that tasks can be classified by.
-  # You can also think of strings here as candidates for formal metadata,
-  # e.g. `purpose: 'build' || 'test'`.
-  tags: JSON
-
-  # Object with properties that can hold any kind of extra data that should be
-  # associated with the task. This can be data for the task which doesn't fit
-  # into `payload`, or it can be supplementary data for use in services
-  # listening for events from this task. For example this could be details to
-  # display on _Treeherder_, or information for indexing the task. Try to put
-  # all related information under one property, so `extra` data keys for
-  # Treeherder reporting and task indexing don't conflict.
-  # **Warning**, do not stuff large datasets in here.
-  # Task definitions should not take-up multiple MBs.
-  extra: JSON
-
-  # If present, these options are passed to the taskcluster-client's `use' method
-  # https://github.com/taskcluster/taskcluster/tree/main/clients/client#calling-api-end-points
-  # before calling queue.createTask.  This is typically used to specify
-  # `options.authorizedScopes`.  Note that the supplied options are not
-  # considered part of the task definition, and this value is only seen in calls
-  # to the createTask mutation.
-  options: JSON
-}
-
```

Comment:
Happy to see this duplication removed!

Are there any side effects of allowing an arbitrary blob (such as possible abuses)? I haven't really understood what role GraphQL plays when our client libraries are generated based on our schemas and references. Is GraphQL used just by the tools UI for the web client?